src/Pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
```

Comment:
Please create `AdminTemplate` in `layouts`. This template will be used for pages in the admin zone. So we need only general structure and styles. All content should be passed by props.